_site/posts/posts.json

```diff
@@ -1,4 +1,21 @@
tterns that are inefficient, hard to read, or likely bugs.\nJarl can parse dozens of files and thousands of lines of code in milliseconds.\nHere is an example of Jarl running on r-source (approximately 1000 files and 200k lines of R code) in about 700 milliseconds:\n\nOn top of that, Jarl can apply automatic fixes2:\n\nJarl is built on the shoulders of giants, in particular:\nlintr: this package provides dozens of rules for linting R code, and Jarl wouldn’t exist without this package. Jarl currently supports 25 lintr rules.\nAir: this is a fast R formatter written in Rust, developed by Lionel Henry and Davis Vaughan, and released earlier this year. This is the technical foundation on which Jarl is built, since Air provides the infrastructure to parse and manipulate R code.\nJarl is a single binary, meaning that it doesn’t need an R installation to work.\nThis makes it a very attractive option for continuous integration for instance, since it takes less than 10 seconds to download the binary and run it on the repository.\nUsing Jarl\nThere are two ways to use Jarl:\nvia the terminal, using jarl check [OPTIONS];\nusing the integration in your coding editor (at the time of writing, only VS Code and Positron have a Jarl extension, but support for Zed is coming soonTM).\nThe Jarl extension enables code highlighting and quick fixes.\nThe former means that code that violates any of the rules in your setup (more on this below) will be underlined and will show the exact violation when hovered.\n\nThe latter adds a lightbulb button next to rule violations, allowing you to selectively apply fixes.\nConfiguring Jarl\nBy default, Jarl will report violations for almost of its rules.\nIt is possible to configure its behavior using a configuration file named jarl.toml.\nIn particular, in this file, you can specify:\nthe rules you want to apply\nthe files to include or exclude\nthe rules for which you want to apply automatic fixes\nand more.\nConclusion\nJarl is in its early days, there are more rules and options to add.\nStill, it can already be used in interactive use or in continuous integration (check out the setup-jarl workflow!).\nThis was a very light introduction, go to the Jarl website for more information.\nIf you want to help developing Jarl, check out the “Contributing” page.\nJarl is written in Rust, which may be a barrier to contributing but is also a very powerful language which is a real pleasure to use.\nI will add a more detailed tutorial soon so that this can also be a nice introduction to this language.\nAcknowledgements\nAs I said above, Jarl depends enormously on the work of lintr and Air developers, so thank you!\nJarl is also very inspired by similar tools in other languages, in particular Ruff in Python and Cargo clippy in Rust.\nFinally, thanks to the R Consortium for funding part of the development of Jarl via the ISC Grant Program.\n\nJarl stands for “Just Another R Linter”.↩︎\nThis is not always possible, it depends on the rule.↩︎\n",
```

Comment:
I am adding comment not necessarily in a place that makes sense. :smile_cat: 

When you write "lintr is a package", you should write "lintr is an R package" to make it even clearer what it is.